client/packages/common/src/ui/components/domain/Dashboard/StatsPanel.tsx

```diff
@@ -126,7 +129,7 @@ export const StatsPanel: FC<StatsPanelProps> = ({
             color="secondary"
             style={{ fontSize: 12, fontWeight: 500 }}
           >
-            {title}
+            {link ? <StyleFreeLink href={link}>{title}</StyleFreeLink> : title}
```

Comment:
StatsPanel is also used in other parts of the app (not in the dashboard), so this link won't always be used.